injective_functions/staking/staking_schema.json

```diff
@@ -17,6 +17,20 @@
           },
           "required": ["validator_address", "amount"]
         }
+      },
```

Comment:
Can you consolidate the schemas into one location? Looks like this one shows up in 3 different schema locations.

Reply:
Agreed, I’ve placed it only inside staking_schema.json.